src/SDKs/BotService/Management.BotService/generate.cmd

```diff
@@ -0,0 +1,7 @@
+﻿::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
```

Comment:
Please create a generate.ps1 file similar to this and regenerate the code.
This should also generate a .txt file that needs to be committed.

Reply:
Will do.